src/App.js

```diff
@@ -1,27 +1,33 @@
-import logo from './logo.svg';
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
+// import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// export default function App() {
+//   return (
+//     <Calculator />
+//   );
+// }
+
+class App extends React.Component {
+  render() {
+    return (
+      <Calculator />
+    );
+  }
 }
+// function App() {
+//   return (
+//     <div className="App">
+//       {/* <header className="App-header"> */}
+//       <h1>Say hello</h1>
+//       {/* <Calculator /> */}
+//       {/* <Calculator /> */}
+//       {/* </header> */}
+//       {/* <Calculator /> */}
+//     </div>
+//   );
+// }
```

Comment:
- [ ] This file is literally full of old commented code.  Please, kindly remove it to improve the repo.